EFCore.NamingConventions/Internal/LowerCaseFirstCharacterNameRewriter.cs

```diff
@@ -0,0 +1,18 @@
+using System.Globalization;
+
+namespace EFCore.NamingConventions.Internal
+{
+    public class LowerCaseFirstCharacterNameRewriter : INameRewriter
+    {
+        private readonly CultureInfo _culture;
+
+        public LowerCaseFirstCharacterNameRewriter(CultureInfo culture) => _culture = culture;
```

Comment:
nit: insert empty line between the constructor and the method

Reply:
ok

Reply:
done